### PR TITLE
fix failing pipeline integration tests

### PR DESCRIPTION
**Pull request by bourdakos1**
Notes:
- Doesn't show `save and submit` dialog anymore, is this a regression or expected?
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: tests/support/commands.ts [hunk 1]
```diff
@@ -75,7 +75,35 @@ Cypress.Commands.add(
...
+  }).dblclick();
+});
+
+Cypress.Commands.add('bootstrapFile', (name: string): void => {
+  cy.readFile(`tests/assets/${name}`).then((file: any) => {
+    cy.writeFile(`build/cypress-tests/${name}`, file);
+  });
+});
+
+Cypress.Commands.add('resetJupyterLab', (): void => {
```
Comment (lresende) [inline tests/support/commands.ts:103]: A lot of the other tests are still using the old name.

Comment (bourdakos1) [reply tests/support/commands.ts:103]: oh oops, I wonder why linting didn't catch this...

Comment (bourdakos1) [reply tests/support/commands.ts:103]: oh I guess that would fall under typescript and we don't actually ever compile

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.base.json",
```
Comment (marthacryan) [inline tsconfig.json:2]: What is this tsconfig being used for?

File: package.json [hunk 3]
```diff
@@ -14,10 +14,10 @@
     "eslint:check": "eslint . --ignore-path .gitignore --ext .ts,.tsx,.js",
     "prettier": "prettier --ignore-path .gitignore --write \"**/*{.ts,.tsx,.js,.jsx,.css,.json}\"",
     "prettier:check": "prettier --ignore-path .gitignore --check \"**/*{.ts,.tsx,.js,.jsx,.css,.json}\"",
-    "start-jupyter": "mkdir -p build/cypress-tests && jupyter lab --config=./tests/test-config.py",
     "test": "npm run test:unit && npm run test:integration",
-    "test:integration": "start-server-and-test start-jupyter http-get://localhost:58888?token=test cy:run",
-    "test:integration:debug": "start-server-and-test start-jupyter http-get://localhost:58888?token=test cy:open",
+    "start": "mkdir -p build/cypress-tests && ts-node scripts/start-test-server.ts",
+    "test:integration": "server-test 'yarn start' ':9000/minio/health/live|http-get://localhost:58888?token=test' 'yarn cy:run'",
+    "test:integration:debug": "server-test 'yarn start' ':9000/minio/health/live|http-get://localhost:58888?token=test' 'yarn cy:open'",
```
Comment (marthacryan) [inline package.json:20]: What is this minio url being used for?

File: tsconfig.json (hunk 2 above)
Comment (bourdakos1) [reply tsconfig.json:2]: This is used for the scripts in the `scripts` folder

File: package.json (hunk 3 above)
Comment (bourdakos1) [reply package.json:20]: these are the urls for the liveliness checks before starting the cypress tests. Some of the tests depend on minio so I added it's url here too

**Pull request merged**